### PR TITLE
Changed delay

### PR DESCRIPTION
**Pull request by asennain**

### CODE REVIEW

**Comment by asennain** [comment]
Related to issue #9

File: src/geldaq/Abdul's Shenanigans/TestCodeJnotebook.ipynb [hunk 1]
```diff
@@ -31,7 +31,7 @@
     "# Will write function for this in future, set the delay as a parameter (Convert to ms)\n",
     "# for use as a function parameter ex. delay = 150\n",
     "\n",
-    "delay = 0.01\n",
+    "delay = 0.03\n",
```
Comment (blakeNaccarato) [inline src/geldaq/Abdul's Shenanigans/TestCodeJnotebook.ipynb:34]: Here's a toy example of how I might request changes to a section of your code. This is trivial, but perhaps you could change this delay to 0.05 instead?

**Review by blakeNaccarato — CHANGES_REQUESTED** [review]
Here you can see how a Pull Request review works. I have commented on parts of your code and am recommending certain changes before we can merge this code.

**Comment by blakeNaccarato** [comment]
Now that I have reviewed your work and requested changes as above, you can make further changes in the same branch (e.g. `Abdul_Changes`) and push them, then they will appear here and we can revisit the changes, and eventually merge.

File: src/geldaq/Abdul's Shenanigans/TestCodeJnotebook.ipynb (hunk 1 above)
Comment (asennain) [reply src/geldaq/Abdul's Shenanigans/TestCodeJnotebook.ipynb:34]: I don't know why but 0.05 sounds like a funny number. I don't like it.

**Pull request merged**